### PR TITLE
Lab4

### PR DESCRIPTION
**Pull request by TlucasX5**
week 4 session 7
student roster

### CODE REVIEW

**Comment by TlucasX5** [comment]
not sure how this "rename" file got in here... how do i remove this?

File: labs/lab-4/studentroster.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    let s1 = (sl1 + ", " + sf1);
+    let s2 = (sl2 + ", " + sf2);
+    let s3 = (sl3 + ", " + sf3);
+    let s4 = (sl4 + ", " + sf4);
+    let s5 = (sl5 + ", " + sf5);
+
+    let result = ("Last Name, First Name\n----------------------\n" + s1 + "\n" + s2 + "\n" + s3 + "\n" + s4 + "\n" + s5);
+
+    const res = (result);
```
Comment (akennis) [inline labs/lab-4/studentroster.js:26]: not necessary (but fine)

File: labs/lab-4/studentroster.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const sl3 = document.getElementById('lname3').value;
+
+    const sf4 = document.getElementById('fname4').value;
+    const sl4 = document.getElementById('lname4').value;
+
+    const sf5 = document.getElementById('fname5').value;
+    const sl5 = document.getElementById('lname5').value;
+
+
+    let s1 = (sl1 + ", " + sf1);
```
Comment (akennis) [inline labs/lab-4/studentroster.js:18]: try using string interpolation where it might be helpful

File: labs/lab-4/studentroster.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    let s2 = (sl2 + ", " + sf2);
+    let s3 = (sl3 + ", " + sf3);
+    let s4 = (sl4 + ", " + sf4);
```
Comment (akennis) [inline labs/lab-4/studentroster.js:21]: these could all have been `const` too - not a problem just a suggestion

**Review by akennis — APPROVED** [review]
100%

**Pull request merged**